protocol/typescript/src/solarxr-protocol/datatypes/body-part.ts

```diff
@@ -25,5 +25,7 @@ export enum BodyPart {
   RIGHT_HAND = 19,
   LEFT_SHOULDER = 20,
   RIGHT_SHOULDER = 21,
-  UPPER_CHEST = 22
+  UPPER_CHEST = 22,
+  LEFT_HIP = 23,
+  RIGHT_HIP = 24
```

Comment:
does it let you put a comma at the end?
```suggestion
  RIGHT_HIP = 24,
```

Reply:
That's the typescript code, the flatbuffer schema file does have it

Reply:
oop I am blind thanks